src/zarr/api/asynchronous.py

```diff
@@ -1065,20 +1065,15 @@ async def create(
 async def empty(
     shape: ChunkCoords, **kwargs: Any
 ) -> AsyncArray[ArrayV2Metadata] | AsyncArray[ArrayV3Metadata]:
-    """Create an empty array.
+    """Create an empty array with the specified shape. The contents will be filled with the
+    array's fill value or zeros if no fill value is provided.
 
     Parameters
     ----------
     shape : int or tuple of int
         Shape of the empty array.
     **kwargs
         Keyword arguments passed to :func:`zarr.api.asynchronous.create`.
-
-    Notes
-    -----
-    The contents of an empty Zarr array are not defined. On attempting to
```

Comment:
A bit of a random thought, but this sentence could likely stay in the sense that not defined can also mean "with `fill_value OR 0`" if we are concerned that this isn't a constraint we want to impose in the future.